qa-pipelines/README.md

```diff
@@ -109,3 +110,21 @@ The composability of the pipeline tasks means there are some interesting things
 In order to do this, replace the CAP bundle URLs in the pipeline config, enable *only* the pre-upgrade tests, and deploy the pipeline. The deploy will use the bundle from the `cap-sle-url` or `cap-opensuse-url` settings in the pipeline config.
 ## Continue a test suite from where a previous build left off.
 Sometimes running tests may fail for timing-related reasons which may be intermittent. If this happens, and you want to try to re-run the test and continue the build from where it left if, you can deploy a new pipeline with only the failed test and following tasks enabled, unlock the config which was used, and run a build from the new pipeline
+
+
+# Dev Nightly Upgrades CI
+
+Nightly Builds are builds which happen every night from develop branch of scf. The build lands in s3://cap-release-archives/nightly/
```

Comment:
```
... and will be picked up by any unpaused pipeline deployed with the `--nightly` flag with an available pool resource
```

Reply:
Dev nightly upgrades CI has nothing to do with any pool, other than capci